File: PASCAL/Cap05/c05ex09.pas
program c05ex09;

label 
  loop, endloop;
  
var
  I, N, R: Integer;

begin
  I := 1;
  loop:
    Write('Entre um valor numerico: ');
    ReadLn(N);

    R := N * 3;

    WriteLn('Resultado = ', R);
    WriteLn;

    if (I > 4) then goto endloop;

    I := I + 1;
    goto loop;
  endloop:

  WriteLn;
  Write('Tecle <Enter> para encerrar... ');
  ReadLn;
end.

File: C/Cap05/c05ex09.c
// c05ex09.c

#include <stdio.h>

int main(void) {
  int I, N, R;

  I = 1;
  loop:
    printf("Entre um valor numerico: ");
    scanf("%d", &N);
    while (getchar() != '\n');

    R = N * 3;

    printf("Resultado = %d\n\n", R);

    if (I > 4) goto endloop;
    ++I;

    goto loop;
  endloop:

  printf("\n");
  printf("Tecle <Enter> para encerrar... ");
  while (getchar() != '\n'); 

  return 0;
}
